edge/fonts.scss

```diff
@@ -0,0 +1,6 @@
+@font-face {
+	font-family: 'Batch';
+	src: url('../lib/fonts/batch-icons-webfont.woff') format('woff');
```

Comment:
Does `!url-loader!../lib/fonts/batch-icons-webfont.woff` work here? (to avoid special-casing the build script)